test-result-summary-client/src/Settings/Settings.jsx

```diff
@@ -228,7 +285,7 @@ export default class Settings extends Component {
                     return (
                         this.state.data.length > 0 ?
                             (
-                                <Popconfirm title="Sure to delete?" onConfirm={() => this.onDelete( record.key )}>
+                                <Popconfirm title="Delete it from view and DataBase?" onConfirm={() => this.onDelete( record )}>
```

Comment:
DataBase -> Database

Reply:
Instead of Cancel and OK, please use Delete and Cancel. And we should highlight the Cancel button